docs/sidebars.js

```diff
@@ -485,115 +485,182 @@ module.exports = {
       link: {
         type: "generated-index",
         title: "Core Resources",
-        slug: "/apis/resources/",
+        slug: "/apis/apis/",
         description: "Resource based API definitions",
```

Comment:
I think we should add a more descriptive description here.
What does resource based API mean?
When should I use this one?

Maybe we can even link to a more detailed page, where we describe when to use what